docs/conf.py

```diff
@@ -226,10 +226,20 @@
 # The short X.Y version.
 version = ".".join(release.split(".")[:2])
 
-# Only include dev docs in dev version.
+# Only include dev docs in dev version but keep development/index.rst
 dev = "dev" in release
 if not dev:
-    exclude_patterns += ["development/*", "testhelpers.rst"]
+    exclude_patterns += [
+        "development/a*",
+        "development/b*",
+        "development/c*",
+        "development/d*",
+        "development/r*",
+        "development/s*",
+        "development/t*",
+        "development/v*",
+        "development/w*",
+    ]
```

Comment:
Seems a bit fragile... Maybe instead we can move `development/index` to `index_dev.rst` ?

Reply:
I can try... I remember pydata-sphinx-theme left/right menu being highly influenced by the layout but I don't remember this moving the index to this folder was part of that or not.